### PR TITLE
ADCM-2455_2 add tests for Admin Roles Page

### PR DESCRIPTION
**Pull request by annademina**

### CODE REVIEW

**Review by skhomuti — APPROVED** [review]

File: tests/ui_tests/app/page/admin/page.py [hunk 1]
```diff
@@ -32,85 +34,124 @@
...
+        page = AdminUsersPage(self.driver, self.base_url)
+        page.wait_page_is_opened()
+        return page
+
+    @allure.step('Open Admin Roles page by left menu item click')
+    def open_roles_menu(self) -> "AdminRolesPage":
+        """Open Admin Roles page by menu object click"""
+
+        self.find_and_click(ObjectPageMenuLocators.roles_tab)
+        self.find_and_click(ObjectPageMenuLocators.roles_tab)
```
Comment (Sealwing) [inline tests/ui_tests/app/page/admin/page.py:116]: Why do we need to click it twice? Please add comment

File: tests/ui_tests/test_admin_page.py [hunk 2]
```diff
@@ -20,140 +22,183 @@
...
+    def test_settings_filter(self, settings_page: AdminSettingsPage):
+        """Apply different filters on Admin Settings page"""
+        params = {
+            'search_text': 'ADCM',
+            'field_display_name': "ADCM's URL",
+            'group': 'Global Options',
+        }
+        get_rows_func = settings_page.config.get_all_config_rows
+        with allure.step(
+            f'Search {params["search_text"]} and check {params["field_display_name"]} ' 'is presented after search'
```
Comment (Sealwing) [inline tests/ui_tests/test_admin_page.py:82]: it can be one line, I guess, linter probably will find it

**Review by Sealwing — CHANGES_REQUESTED** [review]

File: tests/ui_tests/test_admin_page.py (hunk 2 above)
Comment (annademina) [reply tests/ui_tests/test_admin_page.py:82]: it is the way black fix it

File: tests/ui_tests/app/page/admin/page.py (hunk 1 above)
Comment (annademina) [reply tests/ui_tests/app/page/admin/page.py:116]: yep, removed

File: tests/ui_tests/test_admin_page.py [hunk 3]
```diff
@@ -88,7 +88,7 @@ def test_settings_filter(self, settings_page: AdminSettingsPage):
             settings_page.config.clear_search()
         with allure.step(
             f'Click on {params["group"]} group and check {params["field_display_name"]} '
-            'is not presented after group roll up'
+            f'is not presented after group roll up'
```
Comment (Sealwing) [inline tests/ui_tests/test_admin_page.py:91]: nothing to format here, typo, I guess

Comment (annademina) [reply tests/ui_tests/test_admin_page.py:91]: it is two lines f-string

Comment (Sealwing) [reply tests/ui_tests/test_admin_page.py:91]: but in the second there's nothing to format or am I missing something?

Comment (annademina) [reply tests/ui_tests/test_admin_page.py]: removed

**Pull request merged**